VTCareService/src/main/java/edu/vt/cs/vtcare/vtcareservice/db/VTCareJDBC.java

```diff
@@ -0,0 +1,38 @@
+package edu.vt.cs.vtcare.vtcareservice.db;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public final class VTCareJDBC {
+    private static VTCareJDBC vtCareJDBC;
+    private Connection con = null;
+
+    private VTCareJDBC() {
+
+    }
+
+    public static VTCareJDBC getInstance() {
+        if (vtCareJDBC == null) {
+            vtCareJDBC = new VTCareJDBC();
+        }
+        return vtCareJDBC;
+    }
+
+    public Connection getConnection() throws Exception {
+        String url = "jdbc:mysql://localhost:3306/VTCareDB";
```

Comment:
These statements can be moved to central config files.

Reply:
Not needed, that is done when there are a lot of properties accessed in the system. Usually we store these with the class itself.